src/Microsoft.DotNet.VersionTools.net45/project.json

```diff
@@ -0,0 +1,16 @@
+{
+  "dependencies": {
+    "Newtonsoft.Json": "7.0.1",
+    "NuGet.Versioning": "3.5.0-beta2-1456",
+    "System.Collections.Immutable": "1.1.37",
+    "System.Reflection.Metadata": "1.1.0",
+    "Microsoft.NETCore.Platforms": "1.0.1-rc2-24027",
+    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0-rc3-24128-00"
```

Comment:
It looks like you're on iteration 2 (old commit), I changed this to 4.0.0 now. (And lots of other dependencies are RTM now too.)

---
In reply to: [70108692](https://github.com/dotnet/buildtools/pull/863#discussion_r70108692) [](ancestors = 70108692)